docs/en/faq/integration/file-export.md

```diff
@@ -9,7 +9,7 @@ toc_priority: 10
 
 ## Using INTO OUTFILE Clause {#using-into-outfile-clause}
 
-Add an [INTO OUTFILE](../../sql-reference/statements/select/into-outfile) clause to your query.
+Add an [INTO OUTFILE](../../../sql-reference/statements/select/into-outfile) clause to your query.
```

Comment:
```suggestion
Add an [INTO OUTFILE](/docs/en/sql-reference/statements/select/into-outfile.md) clause to your query.
```

Reply:
Thanks for the PR! 

links should always start with `/docs` and include the file extension (usually `.md`).  Having the `.md` causes Docusaurus to check for broken links, and having the `/docs` helps avoid 404's on page refresh (no idea what is causing that)